spec/yaml/swiftnav/sbp/imu.yaml

```diff
@@ -69,6 +69,127 @@ definitions:
             type: s16
             desc: Angular rate around IMU frame Z axis
 
+ - MSG_IMU_HIGH_RATE_DATA:
+    id: 0x0905
+    short_desc: IMU data
+    desc: >
+      This message contains timestamped high rate IMU data, i.e. acceleration and angular rate, as well as high-rate
+      status and health information.
+
+      If multiple IMUs are being sent from the same sender ID, they shall use different imu_id values.
+      When the physical IMU changes that's the source of the data (e.g. after a replacement of an ECU), then this
+      shall also result in using a different imu_id for the new IMU.
+
+      Timestamps can either be the GNSS time of week (wrapping around to zero after 604800 seconds) or a monotonic
+      local time (wrapping around to zero after 2^32 milliseconds). The timestamping mode shall be reported as part
+      of the status flags. When monotonic, local time is being used, all IMUs from the same sender ID must use the
+      same monotonic clock and the relation to GNSS time needs to be established using either MSG_GNSS_TIME_OFFSET
+      or MSG_PPS_TIME fom the same sender ID.
+
+      The time-tagging mode should not change throughout a run.
+    fields:
+        - time:
+            type: u32
+            units: ms
+            desc: >
+              Milliseconds since reference epoch.
+        - time_fractional:
+            type: u8
+            units: ms / 256
+            desc: >
+              Milliseconds since reference epoch, fractional part
+        - acc_x:
+            type: s32
+            units: 0.000000228406861 m/s^2
+            desc: Acceleration in the IMU frame X axis
+        - acc_y:
+            type: s32
+            units: 0.000000228406861 m/s^2
+            desc: Acceleration in the IMU frame Y axis
+        - acc_z:
+            type: s32
+            units: 0.000000228406861 m/s^2
+            desc: Acceleration in the IMU frame Z axis
+        - gyr_x:
+            type: s32
+            units: 0.000003725290298 deg/s
+            desc: Angular rate around IMU frame X axis
+        - gyr_y:
+            type: s32
+            units: 0.000003725290298 deg/s
+            desc: Angular rate around IMU frame Y axis
+        - gyr_z:
+            type: s32
+            units: 0.000003725290298 deg/s
```

Comment:
For the documentation, it will be easier to specify the fraction instead of the decimal value